src/DynamoCore/Core/NodeModelAssemblyLoader.cs

```diff
@@ -228,12 +228,12 @@ internal static bool ContainsNodeModelSubType(Assembly assem)
                     //and have the elementname attribute
                     if (IsNodeSubType(t))
                     {
-                        //if we are running in revit (or any context other than NONE) use the DoNotLoadOnPlatforms attribute, 
+                        //if we are running in revit (or any context other than Standalone) use the DoNotLoadOnPlatforms attribute, 
                         //if available, to discern whether we should load this type
-                        if (context.Equals(Context.NONE)
+                        if (context.Equals(DynamoAppContext.DynamoStandalone)
                             || !t.GetCustomAttributes<DoNotLoadOnPlatformsAttribute>(false)
```

Comment:
By the way, I couldn't find any `DoNotLoadOnPlatformsAttribute` in use. I tried searching in Dynamo and Dynamo for Revit, but didn't find anything. It's strange...